assets/css/base/_base.scss

```diff
@@ -783,12 +785,14 @@ table {
 	}
 
 	.entry-content {
-		a:not( .button ):not( .components-button ) {
+		a {
 			text-decoration: underline;
 
 			&:hover {
 				text-decoration: none;
 			}
+
+			@include notUnderlinedWhenButton;
```

Comment:
If we typically add the `()` to all mixins might be nice to do so here.

Reply:
Done in 97894ccfa67b3367a7535117b89afb7a1e978f6d.